exporter/stackdriverexporter/observability.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2020 OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stackdriverexporter
+
+import (
+	"context"
+	"strconv"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// TODO: re-think if processor should register it's own telemetry views or if some other
+// mechanism should be used by the collector to discover views from all components
+
+func init() {
+	view.Register(viewPointCount)
```

Comment:
Will this view be picked up by [self-observability telemetry](https://github.com/open-telemetry/opentelemetry-collector/blob/master/service/telemetry.go) exposing metrics via Prometheus endpoint?

It's not obvious from this code where this view is being exported. Maybe add a link / comment about that?